modules/apps/object/object-rest-test/src/testFunctional/macros/AccountRoleAPI.macro

```diff
@@ -0,0 +1,39 @@
+definition {
+
+	macro associateAccountRoleUserAndAccount {
```

Comment:
the name is not entirely clear to me, what about changing it to something like `associateUserAccountAndRoleToAccount`?

Reply:
I just think of that this method can't relate userAccount to account, it can only relate User which AccountRole is assigned to Account, how about keep the method name associateAccountRoleUserAndAccount?

Reply:
in this case `associateUserToAccountWithAccountRole`?